src/applications/personalization/profile/tests/e2e/direct-deposit/gating.cypress.spec.js

```diff
@@ -74,33 +74,40 @@ function confirmDirectDepositIsBlocked() {
 }
 
 describe('Direct Deposit', () => {
+  let getPaymentInfoStub;
   beforeEach(() => {
     disableFTUXModals();
+    getPaymentInfoStub = cy.stub();
     cy.login();
     cy.route('GET', '/v0/feature_toggles*', dd4eduEnabled);
   });
   it('should be blocked if the user is not in EVSS and they are not signed up for DD4EDU', () => {
     cy.route('GET', 'v0/user', mockUserNotInEVSS);
     cy.route('GET', 'v0/profile/ch33_bank_accounts', mockDD4EDUNotEnrolled);
+    cy.intercept('v0/ppiu/payment_information', () => {
```

Comment:
This mocks the `GET payment_information` endpoint. If our app hit that endpoint, it would return nothing but it _would_ call our stub. So we can check to make sure the stub is never called, which means our app never tried to call the API.

Reply:
Fancy! Learned something new here.